Lib/http/client.py

```diff
@@ -241,10 +243,10 @@ def _parse_header_lines(header_lines, _class=HTTPMessage):
     hstring = b''.join(header_lines).decode('iso-8859-1')
     return email.parser.Parser(_class=_class).parsestr(hstring)
 
-def parse_headers(fp, _class=HTTPMessage):
+def parse_headers(fp, _class=HTTPMessage, _max_headers=None):
```

Comment:
I suggest the following because without keywords we would be stuck with this parameter for ever and deprecation would be very hard to do.

```py
def parse_headers(fp, _class=HTTPMessage, *, _max_headers=None):
```